packages/flutter/lib/src/widgets/fade_in_image.dart

```diff
@@ -137,6 +158,10 @@ class FadeInImage extends StatelessWidget {
     this.alignment = Alignment.center,
     this.repeat = ImageRepeat.noRepeat,
     this.matchTextDirection = false,
+    int placeholderCacheWidth,
```

Comment:
Why not use two Size arguments instead of four ints?

Reply:
Ah, I see. The other Images also expose it as separate ints. I guess we should be consistent.